README.md

```diff
@@ -86,13 +86,15 @@ Rich text elements in Kentico Kontent can contain links to other content items.
 
 ### How to set up webhook-enabled caching
```

Comment:
20 seconds is good for demonstration purposes, but it is not good for caching in a production environment. Please consider changing this value to 10 minutes.